### PR TITLE
Ross documentation

### PR DESCRIPTION
**Pull request by Lawreros**
Documentation of native and native-dsn pipelines, good foundation for further documentation

### CODE REVIEW

File: tests/data/BNU1/sub-0025864/ses-1/.gitattributes [hunk 1]
```diff
@@ -0,0 +1,6 @@
+anat/sub-0025864_ses-1_T1w.nii.gz filter=lfs diff=lfs merge=lfs -text
```
Comment (loftusa) [inline tests/data/BNU1/sub-0025864/ses-1/.gitattributes]: These test files were deleted in the current staging (they now live here: https://github.com/neurodata/neuroparc/tree/master/data/BNU1/sub-0025864/ses-1!)

**Review by loftusa — CHANGES_REQUESTED** [review]
Great job, these changes all look awesome!
Only two things are necessary before I approve:

1) Merge with current staging (has travis updates)
2) Travis showing us that nothing breaks that didn't break before (at least up until the point where Travis currently times out)

File: tests/data/BNU1/sub-0025864/ses-1/.gitattributes (hunk 1 above)
Comment (Lawreros) [reply tests/data/BNU1/sub-0025864/ses-1/.gitattributes]: Manually deleted /ndmg/tests/data

**Comment by dPys** [comment]
Conflicting commit changes resolved.

**Review by loftusa — APPROVED** [review]
Ok, all look good, approved :)

**Pull request merged**